src/main/resources/application.yaml

```
@@ -27,4 +27,9 @@ spring:
 
 logging:
   level:
-    root: info
\ No newline at end of file
+    root: info
+
+jwt:
+  access-token:
+    secret: ${JWT_ACCESS_SECRET:fA7qL8pX9vE3+gH1jZ5rK2cW0uB4dF6yG8iN/oP3tS4=}
+    expiration: ${JWT_ACCESS_EXPIRATION:600000}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)